core/config/config.go

```diff
@@ -23,10 +23,20 @@ type HeartConfig struct {
 	Db         DbConfig                   `toml:"db"`
 	Connection p2ptypes.ConnectionsConfig `toml:"connection"`
 
+	LogDNA LogDNA `toml:"log_dna"`
+
 	// Key to decrypt data sent over network.
 	AesKey []byte
 }
 
+type LogDNA struct {
```

Comment:
Can you move this to `lib` repo? Otherwise this will be duplicated multiple times.